lib/auth/apiserver.go

```diff
@@ -572,6 +604,15 @@ func (s *APIServer) getDomainName(w http.ResponseWriter, r *http.Request, p http
 	return domain, nil
 }
 
+// getU2fAppId returns the U2F AppID in the auth configuration
+func (s *APIServer) getU2fAppId(w http.ResponseWriter, r *http.Request, p httprouter.Params) (interface{}, error) {
+	appid, err := s.a.GetU2fAppId()
```

Comment:
Should we change cases where u2f is the beginning of a variable name?

For example, change u2fReg to U2FReg